control-plane-operator/controllers/hostedcontrolplane/kas/kms/ibmcloud.go

```diff
@@ -79,18 +72,6 @@ func (p *ibmCloudKMSProvider) GenerateKMSEncryptionConfig() (*v1.EncryptionConfi
 		},
 	}
 
-	// iterate in reverse because highest version key should be used for new secret encryption
-	for i := len(keys) - 1; i >= 0; i-- {
-		configEntry := v1.ProviderConfiguration{
-			KMS: &v1.KMSConfiguration{
-				Name:      fmt.Sprintf("%s%d", ibmKeyNamePrefix, keyVersionKeyEntryMap[keys[i]].KeyVersion),
-				Endpoint:  ibmCloudKMSUnixSocket,
-				CacheSize: ptr.To[int32](100),
-				Timeout:   &metav1.Duration{Duration: 35 * time.Second},
-			},
-		}
-		providerConfiguration = append(providerConfiguration, configEntry)
-	}
 	providerConfiguration = append(providerConfiguration, v1.ProviderConfiguration{
 		Identity: &v1.IdentityConfiguration{},
 	})
```

Comment:
Can probably just inline this w/ the declaration above, rather than doing a separate `append`, for brevity.